examples/meta/src/preprocessor/random_fourier_features.sg.in

```diff
@@ -8,7 +8,8 @@ Features feats_test = create_features(data_test)
 #![create_features]
 
 #![create_preprocessor]
-Transformer preproc = create_transformer("RandomFourierGaussPreproc", seed=12345)
+Kernel gaussiankernel = create_kernel("GaussianKernel", log_width=-0.34657359027997264)
```

Comment:
I think we could rely on the default initialization of the kernel here?

Reply:
Let's not because you yourself said it doesn't make sense to default initialize it. There's also the added benefit to set log_width to that peculiar value(which yields width as 1).

Reply:
yes, in the rff class that doesnt make much sense. But now we would rely on the default of the kernel.
Isn't the default value of the gk so that the width is 1? (I thought that)

Reply:
```AutoValue<float64_t> m_log_width = AutoValueEmpty{};``` this is how its def intialized, not sure if it will yield 1.

Reply:
@gf712 it would be cool if this worked for RFFs as well!
In this case, in the meta example, for now init the kernel to something that looks a bit nicer than this number (but that corresponds to a width that is around 1-ish. Meta examples are user facing and should be a bit more tidy.

Reply:
I see, we could probably get it working so that the width initialisation is triggered when we call `get_width` or `get("log_width")`

Reply:
but there are never any features provided to the kernel in this scenario ... Looks like we have to initialise it manually here. Leave the number then @jonpsy Once Gils PR moving back to width rather than log_width is merged, we can just change it to width=1.
 #5086